internal/utils/grpc.go

```diff
@@ -28,38 +22,3 @@ func GrpcDialWithInsecure(serviceName string, serviceURL string) (*grpc.ClientCo
 
 	return cc, nil
 }
-
-// GrpcDialWithInsecureAndBackoff is a blocking function that creates an insecure
-// gRPC connection to serviceURL with a retry algorithm that on connection failure
-// retries to reconnect. The timer deadline, after which the algorithm should stop
-// trying to reconnect, is taken from the ctx parameter, it's expected that the passed
-// in ctx parameter was created with context.WithTimeout. If no deadline is supplied the
-// reconnecting algorithm runs in a loop forever until the service is available again.
-// After successfully connected, any RPC calls made from this connection also have a retry
-// policy of 1 minute after which an error is returned that it couldn't connect to the service.
-func GrpcDialWithInsecureAndBackoff(ctx context.Context, serviceName, serviceURL string) (*grpc.ClientConn, error) {
```

Comment:
Are we no longer using this function?

Reply:
No, was catched by the linter as unused.